src/core/bpf-pinned-prog.h

```diff
@@ -0,0 +1,6 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+typedef struct Unit Unit;
```

Comment:
why not `#include "unit.h"`?

Reply:
Forward declaration habit. Switched to `#include "unit.h"`.